x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/generate_api_key_panel.tsx

```diff
@@ -37,7 +37,7 @@ export const GenerateApiKeyPanel: React.FC = () => {
 
   const cloudContext = useCloudDetails();
 
-  const DEFAULT_URL = '<ELASTICSEARCH-HOST>:<ELASTICSEARCH-PORT>';
+  const DEFAULT_URL = 'https://localhost:9200';
```

Comment:
By default, Elasticsearch starts on the non-secure protocol `http`, right?

```
$ curl -uelastic:changeme 'http://localhost:9200/_cat/indices'
green  open .ent-search-engine-documents-test                        FsaMkjvtQyWpJA4KoqFPMA 2 0     0 0    450b    450b
green  open .ds-logs-enterprise_search.api-default-2022.08.10-000001 DANPBKZ-TfCRCNvlzRZFbA 1 0    22 0 135.6kb 135.6kb
yellow open .monitoring-ent-search-7.16.2-mb-2022.08.10-000001       1yF2mvW4SOevHDjv51hAXw 1 1 12126 0   8.8mb   8.8mb

$ curl -uelastic:changeme 'https://localhost:9200/_cat/indices'
curl: (35) error:1404B42E:SSL routines:ST_CONNECT:tlsv1 alert protocol version
```

Reply:
It should be https if I remember correctly from latest install tests.  https://www.elastic.co/guide/en/elasticsearch/reference/current/targz.html#_check_that_elasticsearch_is_running

Reply:
Guess you gotta use a cert 😅

Reply:
CERTainly :P